### PR TITLE
refacto(escrow): remove useless storage variable

### PR DESCRIPTION
**Pull request by n1c01a5**

### CODE REVIEW

**Comment by ferittuncer** [comment]
I think this change can be a problem in rare cases where each side pays different arbitration costs because of the timing of arbitration cost change. 

For example: 
Sender pays 500,
Receiver pays 3,
Actual arbitration cost when dispute raised is 500,
Transaction amount is 10.

So we collected 513 units in the pool and spent 500 units for arbitration cost, leaves 13 units which should be returned. However `transaction.senderFee + transaction.amount` will make 510 units. So the contract will lose the difference. If arbitrator is malicious, arbitrationCost can be set optimally so that it drains the contract.

In short, this opens up a possibility to extract the difference between senderFee and receiverFee from the contract. So I think the refund should be done considering what was the actual arbitration cost was or it should not be possible to pay different amounts from each side. @n1c01a5

**Comment by ferittuncer** [comment]
All good. @n1c01a5
